lib/commands/push.ts

```diff
@@ -373,6 +382,7 @@ export default class PushCmd extends Command {
 			auth: token,
 			baseUrl,
 			nogitignore: !options.gitignore,
+			isdraft: options.draft,
```

Comment:
I think `isdraft: options.draft` should be move a few lines above to the `opts` object instead of the `args` object, and also be listed in BuildOpts here:
https://github.com/balena-io/balena-cli/blob/v12.44.16/lib/utils/remote-build.ts#L37-L44
and then also be listed in this query string block:
https://github.com/balena-io/balena-cli/blob/v12.44.16/lib/utils/remote-build.ts#L89-L95

Reply:
Done! Thanks for that!